docresources/SyncBot/syncbot-readme.md

```diff
@@ -315,6 +315,30 @@ and someone with write access to this repo to perform the final step
 		<checked>
 	```
 
+### Setting up a Github org for SyncBot
+
+The process for SyncBot is similar to
+[VersionBot](https://github.com/resin-io-modules/resin-procbots/blob/master/docresources/VersionBot/versionbot.md#creating-a-github-integration)
+with the following exceptions:
```

Comment:
This forces the user to go read separate directions, and then merge in a patch set (effectively) from below. If versionbot changes its instructions, it could break syncbot installs. I think you should just copy the directions here, or make a 3rd doc which has basic instructions, and then you switch to the bot-specific ones to finish.